pkg/testing/cli.go

```diff
@@ -84,8 +94,10 @@ Read more at cartographer.sh`,
 }
 
 func init() {
-	rootCmd.Flags().StringVarP(&directory, "directory", "d", "", "directory to test")
-	rootCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "output logs and increase test failure verbosity")
+	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "output logs and increase test failure verbosity")
+
+	rootCmd.AddCommand(templateCmd)
+	templateCmd.Flags().StringVarP(&directory, "directory", "d", "", "directory to test")
 
-	_ = rootCmd.MarkFlagRequired("directory")
+	_ = templateCmd.MarkFlagRequired("directory")
```

Comment:
it might be worth also marking the directory param as a "directory flag"

```go
// MarkFlagDirname instructs the various shell completion implementations to
// limit completions for the named flag to directory names.
func (c *Command) MarkFlagDirname(name string) error {
        return MarkFlagDirname(c.Flags(), name)
}
```

ala

```go
	_ = templateCmd.MarkFlagDirname("directory")
```